docs/synapse/deploymentguide.rst

```diff
@@ -0,0 +1,363 @@
+
+Start the container::
+
+    docker-compose --file /srv/syn/00.jsonstor/docker-compose.yaml pull
+    docker-compose --file /srv/syn/00.jsonstor/docker-compose.yaml up -d
+
+Deploy Cortex Service
+=====================
+
+**Inside the AHA container**
+
+Generate a one-time use provisioning URL::
+
+    python -m synapse.tools.aha.provision.service 00.cortex --cellyaml /tmp/cortex.yaml
+
+You should see output that looks similar to this::
+
+    one-time use URL: ssl://aha.loop.vertex.link:27272/<guid>?certhash=<sha256>
+
+**On the Host**
+
+Create the container directory::
+
+    mkdir -p /srv/syn/00.cortex/storage
+    chown -R 999 /srv/syn/00.cortex/storage
+
+Create the ``/srv/syn/00.cortex/docker-compose.yaml`` file with contents::
+
+    version: "3.3"
+    services:
+      00.cortex:
+        user: "999"
+        image: vertexproject/synapse-cortex:v2.x.x
+        network_mode: host
+        restart: unless-stopped
+        volumes:
+            - ./storage:/vertex/storage
+        environment:
+            - SYN_CORTEX_AXON=aha://axon...
+            - SYN_CORTEX_JSONSTOR=aha://jsonstor...
+            - SYN_CORTEX_AHA_PROVISION=ssl://aha.loop.vertex.link:27272/<guid>?certhash=<sha256>
+
+.. note::
+
+    Don't forget to replace your one-time use provisioning URL!
+
+Start the container::
+
+    docker-compose --file /srv/syn/00.cortex/docker-compose.yaml pull
+    docker-compose --file /srv/syn/00.cortex/docker-compose.yaml up -d
+
+Remember, you can view the container logs in real-time using::
+
+    docker-compose --file /srv/syn/00.cortex/docker-compose.yaml logs -f
+
+Deploy Cortex Mirror (optional)
+===============================
+
+**Inside the AHA container**
+
+Generate a one-time use URL for provisioning from *inside the AHA container*::
+
+    python -m synapse.tools.aha.provision.service 01.cortex --mirror cortex
+
+You should see output that looks similar to this::
+
+    one-time use URL: ssl://aha.loop.vertex.link:27272/<guid>?certhash=<sha256>
+
+**On the Host**
+
+Create the container storage directory::
+
+    mkdir -p /srv/syn/01.cortex/storage
+    chown -R 999 /srv/syn/01.cortex/storage
+
+Create the ``/srv/syn/01.cortex/docker-compose.yaml`` file with contents::
+
+    version: "3.3"
+    services:
+      01.cortex:
+        user: "999"
+        image: vertexproject/synapse-cortex:v2.x.x
+        network_mode: host
+        restart: unless-stopped
+        volumes:
+            - ./storage:/vertex/storage
+        environment:
+            # disable HTTPS API for now to prevent port collisions
+            - SYN_CORTEX_HTTPS_PORT=null
+            - SYN_CORTEX_AHA_PROVISION=ssl://aha.loop.vertex.link:27272/<guid>?certhash=<sha256>
+
+.. note::
+
+    Don't forget to replace your one-time use provisioning URL!
+
+Start the container::
+
+    docker-compose --file /srv/syn/01.cortex/docker-compose.yaml pull
+    docker-compose --file /srv/syn/01.cortex/docker-compose.yaml up -d
```

Comment:
Starting the mirror causes the following exception

```
synapse.exc.NoSuchName: NoSuchName: mesg='aha lookup failed: cortex'
```

Super odd since the cortex shows itself as registered.

Reply:
I think i found/fixed this